mubench.reviewsite/tests/RunsControllerTest.php

```diff
@@ -407,6 +407,48 @@ function test_get_misuse_results_no_runs()
         self::assertEquals([], $misuses);
     }
 
+    function test_update_run_timestamp_on_new_finding()
+    {
+        $experiment = Experiment::find(1);
+
+        $this->runsController->addRun($experiment->id, $this->detector1->muid, '-p-', '-v-', [
+            "result" => "success",
+            "runtime" => 42.1,
+            "number_of_findings" => 23,
+            "timestamp" => 12,
+            "potential_hits" => [
+                [
+                    "misuse" => "-m-",
+                    "rank" => 0,
+                    "target_snippets" => [
+                        ["first_line_number" => 5, "code" => "-code-"]
+                    ]
+                ]
+            ]
+        ]);
+        sleep(1);
```

Comment:
Instead of waiting, can we manually change the date and expect this change to be overwritten?